### ISSUE TITLE
Are 'dynamic_feat' and 'cat' field names in input data supported by GluonTS DeepAR?

### ISSUE THREAD

**Issue by sujayramaiah** [awslabs/gluon-ts#1032]
Hi, 

When using GluonTS DeepAR, we used the labels "dynamic_feat" for specifying the dynamic feature series and "cat" for specifying the Categorical columns similar to how we specify when using SageMaker's DeepAR. We are not seeing any errors and we do see specifying different features is producing different results. We are using default hyperparameter settings and only tuning the epoch and learning rate.

When going through the documentation, we see that the support for legacy name for dynamic features as "dynamic_feat" is present and being handled but do not see much information about "cat".

Can you please clarify if both these labels are accepted by gluonTS DeepAR also?

Thanks in advance!

**Comment by jaheba** [comment]
Hey,

I might be wrong, but I don't think we support `dynamic_feat` either (for some reason we use it `src/gluonts/dataset/stat.py`). I'm happy to be corrected on this.

We essentially renamed them, `cat` -> `feat_static_cat` and `dynamic_feat` -> `feat_dynamic_real`.

Closing the issue, but feel free to comment on it.

**Comment by sujayramaiah** [comment]
@jaheba Thanks for the clarification. We will use the new labels `feat_static_cat` and `feat_dynamic_real`.
I did see dynamic_feat as the legacy column in the `gluonts.dataset.field_names` module.
Since we did not receive any errors, how would these columns be used internally? Would they be ignored? Please clarify.
Thanks

**Comment by jaheba** [comment]
Yes, we ignore any additional ￼fields passed.

The fields defined in the fieldnames files are basically just names and not all fields are supported in all algorithms.

**Comment by sujayramaiah** [comment]
Thanks @jaheba !
After updating the field labels to `feat_static_cat` and `feat_dynamic_real`, most of the iterations are giving similar results and not clearly evident if any of the new dynamic features are helping. Can you please advise/point me to any resource which helps us identify if any dynamic features or categorical features are improving the results?
Thanks

**Comment by mavarick** [comment]
I would like to know: why don’t have `feat_dynamic_cat`, if there exists `feat_dynamic_real`?
